hack/install-build.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+install bazel-bin/cmd/clusterctl/darwin_amd64_pure_stripped/clusterctl /usr/local/bin
```

Comment:
If this is mainly intended for local testing and dev workflows, it might be better to target ~/bin rather than /usr/local/bin to avoid the need for privilege escalation.

Reply:
oh, i always thought /usr/local/bin/ was owned by the user, but maybe that's a homebrew/os x thing

Reply:
I'm curious if we thought about overwriting the GOPATH/bin binaries instead?

Reply:
something like go install? :) that would be my ideal solution

Reply:
Yeah I think that'd be a nicer approach than writing to /usr/local